zcash_client_sqlite/src/wallet/db.rs

```diff
@@ -1030,13 +1030,15 @@ WITH unioned AS (
            ro.output_index              AS output_index,
            from_account.uuid            AS from_account_uuid,
            to_account.uuid              AS to_account_uuid,
-           NULL                         AS to_address,
+           a.address                    AS to_address,
```

Comment:
As noted in pairing, documentation of this needs updating because this column is no longer guaranteed to be null for non-external (and instead downstreams need to use `to_account_uuid IS NULL` to identify external.